### PR TITLE
move code out of receiverwrapper

### PR DESCRIPTION
**Pull request by snopoke**
and a small prefactor

@benrudolph @czue 

cc @calellowitz @biyeun 

Tested locally:
- repeaters UI
- submission errors reports
- repeaters tests

### CODE REVIEW

File: corehq/apps/hqwebapp/views.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
 from corehq.apps.dropbox.exceptions import DropboxUploadAlreadyInProgress
 from corehq.apps.hqwebapp.encoders import LazyEncoder
 from corehq.apps.hqwebapp.forms import EmailAuthenticationForm, CloudCareAuthenticationForm
-from corehq.apps.receiverwrapper.models import Repeater
+from corehq.apps.repeaters.models import Repeater
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/views.py]: F401 'Repeater' imported but unused

File: corehq/apps/repeaters/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# following import registers all Generator classes to a Mapping
+import corehq.apps.repeaters.repeater_generators
```
Comment (dimagimon) [inline corehq/apps/repeaters/__init__.py]: F401 'corehq' imported but unused

File: corehq/apps/repeaters/models.py [hunk 3]
```diff
@@ -493,4 +493,4 @@ def fire(self, max_tries=3, post_fn=None):
                     self.update_failure()
 
 # import signals
-from corehq.apps.receiverwrapper import signals
+from corehq.apps.repeaters import signals
```
Comment (dimagimon) [inline corehq/apps/repeaters/models.py]: F401 'signals' imported but unused

**Comment by czue** [comment]
looks good, i think the build fail is real
```
======================================================================
ERROR: test_no_subscription (corehq.apps.reports.tests.test_export_api.ExportTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/tests/test_export_api.py", line 176, in test_no_subscription
    _submit_form(f, community_domain.name)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/tests/test_export_api.py", line 43, in _submit_form
    return spoof_submission(url, f)
  File "/home/travis/build/dimagi/commcare-hq/corehq/util/test_utils.py", line 26, in inner
    return fn(*args, **kwargs)
  File "corehq/ex-submodules/couchforms/util.py", line 530, in spoof_submission
    return response['X-CommCareHQ-FormID']
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django/http/response.py", line 187, in __getitem__
    return self._headers[header.lower()][1]
KeyError: 'x-commcarehq-formid'
----------------------------------------------------------------------
```

**Comment by snopoke** [comment]
:white-check-mark:

**Pull request merged**